libsolidity/analysis/ControlFlowRevertPruner.cpp

```diff
@@ -57,6 +58,7 @@ void ControlFlowRevertPruner::run()
 void ControlFlowRevertPruner::findRevertStates()
 {
 	std::set<CFG::FunctionContractTuple> pendingFunctions = keys(m_functions);
+	std::map<CFG::FunctionContractTuple, std::set<CFG::FunctionContractTuple>> wakeUp;
```

Comment:
This is "tuples to be woken up when the key tuple changes its state from unknown to known", right? Might be worth a small comment for future reference? But also fine without.